Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -280,12 +287,9 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
   [_pendingRequests removeAllObjects];
 
   // Stop updating if not not observing
-  if (!_observingLocation) {
+  if (!!_observingLocation) {
```

Comment:
I thought so as well, but given the _not not_ I thought not using !! in the code was an omission.

So given the awesome feedback, I'll update the PR shortly.